docs/source/usage.rst

```diff
@@ -325,6 +325,27 @@ Example:
     >>> print(type(conf.client.url).__name__)
     str
 
+Interpolated nodes can be any node in the config, not just leaf nodes:
+
+.. doctest::
+
+    >>> cfg = OmegaConf.create(
+    ...     {
+    ...         "john": {"height": 180, "weight": 75},
+    ...         "fred": {"height": 195, "weight": 90},
+    ...         "player": "${john}",
+    ...     }
+    ... )
+    >>> cfg.player.height
+    180
+    >>> cfg.player.weight
+    75
+    >>> cfg.player = "${fred}"
+    >>> cfg.player.height
+    195
+    >>> cfg.player.weight
+    90
```

Comment:
I think this simpler example is sufficient:

```suggestion
    >>> cfg = OmegaConf.create(
    ...     {
    ...         "john": {"height": 180, "weight": 75},
    ...         "player": "${john}",
    ...     }
    ... )
    >>> cfg.player
    {"height": 180, "weight": 75}
```

Reply:
Alright, I took your suggestion!